### ISSUE TITLE
using hash links to access certain slide, the play continue and you cant select that slide again

### ISSUE THREAD

**Issue by alhoseany** [nicinabox/superslides#28]
hi, lets say you access the slide page using the url with hash "#2".
this loads the slideshow and takes you to slide number 3.
the problem here that the slideshow starts playing again and when you try to click on the pagination item corresponds to the 3rd slide , nothing happen. you cant go back to slide 3 unless you wait for it to come in its order of playing.

**Comment by nicinabox** [comment]
Ah, interesting. Yes. The reason is that the url still has that hash, so when you click on that item again, it still things it's on #2 according to the url. Definitely a bug. Nice catch.

**Comment by alhoseany** [comment]
i found a solution to this. just will stop the slideshow when the slideshow is accessed by hash url.

``` javascript
  start = function() {
    var index;
    if (size > 1) {
      if (location.hash) {
        index = location.hash.replace(/^#/, '');
              animate(index);     // animate only
      } else {
        index = (first_load ? 0 : "next");
              animate(index);    // animate and play
              return play();
      }

    } else {
      $container.fadeIn('fast');
      return $("." + $.fn.superslides.options.nav_class).hide();
    }
  };
```

**Comment by alhoseany** [comment]
another solution would be to check for the current slide when the pagination item is clicked.

**Comment by nicinabox** [comment]
This is already being reworked in the 0.5-wip branch. I'll take care of this bug there.

**Comment by nicinabox** [comment]
Fixed in 0.5. Shooting to release this weekend.
